### PR TITLE
DEX: Implement left nav collapse/expand (toggle)

### PR DESCRIPTION
**Pull request by rlucas6130**
## Ticket
* https://issues.aphelion-neo.com/tktview?name=3a8f9ceba7

## Changes
* left nav collapse/expand feature. Enabled at the route level (using the meta field). 
* Default state for DEX is collapsed.
* Note: expand/collapse is scoped to only the DEX page - however can easily be extended to other pages using the isMenuToggleable flag.

## Screenshots

Collapsed:

image

Expanded: 

image

### CODE REVIEW

File: src/renderer/components/Icon.vue [hunk 1]
```diff
@@ -436,12 +448,30 @@ export default {
     }
   }
 
+  .double-arrow-left {
+    path:first-child {
```
Comment (crishellco) [inline src/renderer/components/Icon.vue]: Let’s use the class instead of element.

File: src/renderer/components/Sidebar.vue [hunk 2]
```diff
@@ -1,70 +1,73 @@
...
+    <template v-if="!toggleable || (toggleable && !collapsed)">
+      <div class="header">
+        <aph-icon name="logo-mark"></aph-icon>
+      </div>
+      <div class="menu link-list">
+        <router-link :to="sendInProgress ? '/authenticated/dashboard/confirming': '/authenticated/dashboard'">
+          <span class="icon">
+            <aph-icon name="dashboard"></aph-icon>
+          </span>
+          <span class="label">{{$t('dashboard')}}</span>
```
Comment (crishellco) [inline src/renderer/components/Sidebar.vue]: Spaces between curly braces and expression.

File: src/renderer/components/Icon.vue (hunk 1 above)
Comment (rlucas6130) [reply src/renderer/components/Icon.vue]: sure - was copying previous implementation for consistency. The 'arrow-right' and 'arrow-left' use the element.

File: src/renderer/components/Sidebar.vue (hunk 2 above)
Comment (rlucas6130) [reply src/renderer/components/Sidebar.vue]: done

**Pull request merged**